### PR TITLE
Implements asynchronous access to 5

### PR DESCRIPTION
**Pull request by inxilpro**
In true Node fashion:

```javascript
five.async(function(err, five) {
	// five === 5
});
```

### CODE REVIEW

**Comment by javonharper** [comment]
+1

File: five.js [hunk 1]
```diff
@@ -91,6 +91,12 @@
     return ['Juwan Howard','Ray Jackson','Jimmy King','Jalen Rose','Chris Webber'];
   };
 
+  five.async = function(callback) {
+    process.nextTick(function() {
+      callback(null, 5);
```
Comment (Joezo) [inline five.js]: should call `five()` here

Comment (inxilpro) [inline five.js]: Good point. Fixed :)

**Comment by Joezo** [comment]
I like this, good work!

**Pull request merged**